iot/iot-amqp-adapter/pom.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we need AMQP adapter as part of this pr?

Reply:
Nope. Should be gone now.